packages/web-app-files/src/spaces/views/Spaces.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="oc-p-s">
+    <h2 class="" v-text="$gettext('Spaces')"></h2>
+    <div class="">
+      <span v-text="$gettext('Access all project related files on one place.')"></span>
+      <a href="#" v-text="$gettext('Learn more about spaces.')"></a>
+      <h3 v-text="$gettext('Your spaces')"></h3>
+      <hr class="uk-margin-bottom" />
+    </div>
+    <div class="spaces-list uk-container uk-container-expand uk-padding-remove">
+      <list-loader v-if="loadSpacesTask.isRunning" />
+      <template v-else>
+        <div
+          v-if="!!spaces.length"
```

Comment:
```suggestion
          v-if="spaces.length"
```
Strict check has no benefit here IMO.